sources/packages/backend/apps/db-migrations/src/sql/Announcements/Create-announcements-table.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS sims.announcements (
+    id SERIAL PRIMARY KEY,
+    message_title VARCHAR(100) NOT NULL,
+    message VARCHAR(200) NOT NULL,
+    start_date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+    end_date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW (),
+    target VARCHAR(200) NOT NULL,
+    -- Audit columns
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    creator INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+    SET
+        NULL,
+        modifier INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+    SET
+        NULL
+);
+
+-- ## Comments
+COMMENT ON TABLE sims.announcements IS 'The table for storing system announcements.';
+
+COMMENT ON COLUMN sims.announcements.message_title IS 'Title for the announcement.';
+
+COMMENT ON COLUMN sims.announcements.message IS 'Message text for the announcement.';
+
+COMMENT ON COLUMN sims.announcements.start_date IS 'When the message starts.';
```

Comment:
Suggestion: `Timestamp when the system announcement starts.`

Likewise for the `end_date`.

Reply:
updated